src/components/trade/TradesTable/index.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import { faExchangeAlt } from '@fortawesome/free-solid-svg-icons'
+
+import { Trade, RawOrder } from 'api/operator'
+
+import { DateDisplay } from 'components/common/DateDisplay'
+import { RowWithCopyButton } from 'components/common/RowWithCopyButton'
+import { BlockExplorerLink } from 'components/common/BlockExplorerLink'
+import {
+  getOrderLimitPrice,
+  getOrderExecutedPrice,
+  formatCalculatedPriceToDisplay,
+  formatExecutedPriceToDisplay,
+  formattedAmount,
+} from 'utils'
+import { getShortOrderId } from '../../../utils/operator'
+import { HelpTooltip } from 'components/Tooltip'
+import StyledUserDetailsTable, {
+  StyledUserDetailsTableProps,
+  EmptyItemWrapper,
+} from '../../common/StyledUserDetailsTable'
+import Icon from 'components/Icon'
+import TradeOrderType from 'components/common/TradeOrderType'
+import { Surplus } from './Surplus'
+import { RAW_ORDER } from '../../../../test/data'
+
+const Wrapper = styled(StyledUserDetailsTable)`
+  > thead > tr,
+  > tbody > tr {
+    grid-template-columns: 10rem 4rem repeat(2, 10rem) repeat(2, 14rem) 10rem 10rem 1fr;
+  }
+`
+
+const TxHash = styled.div`
+  max-width: 10rem;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`
+
+function getLimitPrice(trade: Trade, isPriceInverted: boolean): string {
+  if (!trade.buyToken || !trade.sellToken) return '-'
+
+  const calculatedPrice = getOrderLimitPrice({
+    buyAmount: trade.buyAmount,
+    sellAmount: trade.sellAmount,
+    buyTokenDecimals: trade.buyToken.decimals,
+    sellTokenDecimals: trade.sellToken.decimals,
+    inverted: isPriceInverted,
+  })
+
+  return formatCalculatedPriceToDisplay(calculatedPrice, trade.buyToken, trade.sellToken, isPriceInverted)
+}
+
+function getExecutedPrice(trade: Trade, isPriceInverted: boolean): string {
+  if (!trade.buyToken || !trade.sellToken) return '-'
+
+  const order: RawOrder = {
+    ...RAW_ORDER,
```

Comment:
I missed this in the last review.
Please don't use test data inside regular components.
Use them only on unit tests or stories

I understand you need a RawOrder instance as a parameters to `getOrderExecutedPrice` and you don't have that here.
Then I suggest to update the function params to pick only the required fields https://github.com/gnosis/gp-ui/blob/434997a0260c7e8fb663ba675c70bcba01166a81/src/utils/operator.ts#L146

Such as:

```ts
export type GetRawOrderPriceParams = CommonPriceParams & {
  order: Pick<RawOrder, 'executedSellAmount' | '...' >
}
```

Then you can keep the same interface but do not need to build a RawOrder instance.